services/meta/association.md

```diff
@@ -3,29 +3,40 @@ This service allows creating a direct association between a source device (e.g.
 
 This service intends to replace `group` commands in `dev_sys` service.
 
-A controllable device (like a lamp) would have a list of `in_services` that would indicate the services it supports. A controller (like a button) would have a list of `out_services` instead.
-
 #### Service Name
 
 `association`
 
 #### Interfaces
 
-Type | Interface                  | Value type | Description
------|-----------------------     |------------|------------
-in   | cmd.association.add        | object     | Add members to the group
-in   | cmd.association.delete     | object     | Remove association for a single member
-in   | cmd.association.delete_all | string     | Remove associations for all group members
-in   | cmd.association.get_report | string     | Get all members of a group
-out  | evt.association.report     | object     | Response for `get_report`
-
-#### Service Props
-
-Name             | Supported Values           | Description
------------------|----------------------------|-------------
-`in_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | List of services that can be controlled, e.g. on a lamp
```

Comment:
Please let me know whether these in and out services were used in Zigbee

Reply:
Yes, we use it

Reply:
Alright, I reverted this part